movement_primitives/dmp/_dual_cartesian_dmp.py

```diff
@@ -239,8 +245,29 @@ def __init__(self, execution_time=1.0, dt=0.01, n_weights_per_dim=10,
 
         self._init_forcing_term()
 
-        self.alpha_y = 25.0
-        self.beta_y = self.alpha_y / 4.0
+        if isinstance(alpha_y, float):
```

Comment:
duplicate code

Reply:
Please see [9103981](https://github.com/dfki-ric/movement_primitives/pull/59/commits/91039818c8624e4c282220596192a4bb0863932a), and let me know if this works for you?